recipes/edts

```diff
@@ -1,40 +1,29 @@
 (edts
  :fetcher github
- :repo   "tjarvstrand/edts"
+ :repo   "sebastiw/edts"
  :files  ("*.el"
           "COPYING"
           "COPYING.LESSER"
           "Makefile"
           "README.md"
           "start"
+          "start.bat"
+          "rebar.config"
+          ("config"                     "config/*")
 
           ("elisp/edts"                 "elisp/edts/*.el")
           (:exclude                     "elisp/edts/*-test.el")
 
-          ("lib/edts"                   "lib/edts/Makefile")
-          ("lib/edts"                   "lib/edts/rebar")
-          ("lib/edts"                   "lib/edts/rebar.config")
-          ("lib/edts/priv"              "lib/edts/priv/app.config")
+          ("lib/edts/include"           "lib/edts/include/*")
           ("lib/edts/priv"              "lib/edts/priv/dispatch.conf")
           ("lib/edts/src"               "lib/edts/src/*")
 
-          ("plugins/edts_debug"         "plugins/edts_debug/*.el")
-          (:exclude                     "plugins/edts_debug/*-test.el")
```

Comment:
Yes, I can see that, but we should keep the line `(:exclude "plugins/edts_debug/*-test.el")` because we don't want to include the build tests in the package.

Reply:
Done